### ISSUE TITLE
hup_openstack required after 2 Chef runs to get OpenStack actually going

### ISSUE THREAD

**Issue by erhudy** [bloomberg/chef-bcpc#609]
Instances booted using the *Boot from image (creates a new volume)* option are failing to launch from master now.

**Comment by erhudy** [comment]
This appears to have been caused by #576 - adding back the `glance_api_servers` and `glance_api_insecure` options under the `[DEFAULT]` section makes it happy again (so now the settings are duplicated in two places).

**Comment by erhudy** [comment]
I lied. Making that change and building a fresh head node did not actually fix it. The hunt continues.

**Comment by erhudy** [comment]
Seems more like the salient operation is simply doing a full restart of all OpenStack services with `hup_openstack`. Doing this with the duplicated configuration items removed fixed the problem.

**Comment by erhudy** [comment]
tl;dr summary: the changes from #576 did not actually cause an issue. It seems that some service is not actually in a fully operational state even after two Chef runs and `hup_openstack` is still required to bring the whole thing into an operational state.

**Comment by mihalis68** [comment]
I get this too. For now I will assume a rechef of all nodes and then hup_openstack as a final fettling measure is required for an upcoming on-metal build.

**Comment by erhudy** [comment]
I'm going to close this because both bootstrap pathways restart all OpenStack services at the appropriate time to get them functional.
